docs/east_ocr.md

```diff
@@ -0,0 +1,113 @@
+#  Optical Character Recognition with Directed Acyclic Graph
+
+This document demonstrate how to create and use an Optical Character Recognition (OCR) pipeline based on [east-resnet50](https://github.com/argman/EAST) text detection model,
+[CRNN](https://github.com/MaybeShewill-CV/CRNN_Tensorflow) text recognition combined with a custom node implementation.
+
+Using such pipeline, a single request to OVMS can perform a complex set of operations with a response containing
+recognized characters for all detected text boxes. 
+
+## OCR Graph
+
+Below is depicted the graph implementing a complete OCR pipelines. 
+
+![OCR graph](east_ocr.png)
+
+It includes the following Nodes:
+- Model east-resent50 - inference execution which takes the user image as input. It returns two outputs including information about all detected boxes, their location and scores.
```

Comment:
So it is east or resent50 ;)?

Reply:
full model name is east-resnet50

Reply:
Still there is a typo `resent`. If that's the full model name then it is ok